Mentorship_Problems/Problem-3.js

```diff
@@ -0,0 +1,105 @@
+/**
+ 1. Given an array of integers nums, return the number of good pairs. A pair (i, j) is called good if nums[i] == nums[j] and i < j.
+    Input: nums = [1,2,3,1,1,3]
+    Output: 4
+    Explanation: There are 4 good pairs (0,3), (0,4), (3,4), (2,5) 0-indexed.
+
+    Input: nums = [1,1,1,1]
+    Output: 6
+    Explanation: Each pair in the array are good.
+
+    Input: nums = [1,2,3]
+    Output: 0 */
+
+const findNumberOfGoodPairs = (nums) => {
+  let count = 0;
+  for (let i = 0; i < nums.length; i++) {
+    for (let j = i + 1; j < nums.length; j++) {
+      if (nums[i] === nums[j]) {
+        count++;
+      }
+    }
+  }
+  return count;
+};
+
+console.log(findNumberOfGoodPairs([1, 2, 3]));
+
+// 2. Given a number, write a function to calculate the sum of the digits.
+//     arr = 123
+//     output = 6; 1 + 2 + 3 = 6
+
+const calculateSumOfDigits = (num) => {
```

Comment:
This is good. Acceptable solution. Can you also think of a solution without converting the number into string?